packages/@okta/vuepress-site/docs/guides/third-party-risk-integration/test-integration/index.md

```diff
@@ -0,0 +1,89 @@
+
+1. Call the following POST API from the Risk Integration Postman collection: **Partner: API to get the access token** (`{{url}}/oauth2/v1/token`).
+2. Review the response from the call, and copy the `access_token` value to your Postman's `accessToken` environment variable. A sample response follows:
+
+    ```JSON
+    {
+    "token_type": "Bearer",
+    "expires_in": 3600,
+    "access_token": "eyJraWQiOiJPeXd3TmtXSnBuX1ZaQzB4aUZKUlRIRmdvQXJwOXBaSkROZktiZG4wemVBIiwiYWxnIjoiUlMyNTYifQ.eyJ2ZXIiOjEsImp0aSI6IkFULlYwbHNUVVUxT3RIMlotOWhGcXExSlhteEF5ZXBqWVc0YXVLSnlwTjJiRTgiLCJpc3MiOiJodHRwczovL2R1ZmZpZWxkLm9rdGFwcmV2aWV3LmNvbSIsImF1ZCI6Imh0dHBzOi8vZHVmZmllbGQub2t0YXByZXZpZXcuY29tIiwic3ViIjoiMG9hYWFib3l4c2JyV2RzazgxZDYiLCJpYXQiOjE2MTEzNDcwNDAsImV4cCI6MTYxMTM1MDY0MCwiY2lkIjoiMG9hYWFib3l4c2JyV2RzazgxZDYiLCJzY3AiOlsib2t0YS5yaXNrRXZlbnRzLm1hbmFnZSJdfQ.YUYhkfj-vjW2zEWIfhdcqMMONRVUV81gdia1wf3C2HZ7qMG6u8aGsRpdaMBotHOeno3ECQupf_hcWpUOPJ6OJX1Zdycn6ui7nDcIar6JfSs6VoyOf_e4pNnj2iBPEy9_F4qlk08Z4tBPL9XMMzUnFKL3ZfMTspBNFwpzXAlrj_wzhDS2TrE0O2Z5EAQc1hKmx7cbCOPOmhtbHDjB1OYDiKlK1Z2OlXvHLxhHGDAVQaPf8tMMD8gqQQ3_Lxifi55gCv5h3ZfVyrJtfZK5v3ZrfapK1u1JbvjvJ2fvjUce0Lf2Jl0Gq8nwD0SZZTYdDxcwJny0F1rjq_FDulaBc0JrUw",
+    "scope": "okta.riskEvents.manage"
+    }
+    ```
+    >**Note**: The access token expires in 60 minutes (or the value set in the property `expires_in`).
+
+For further background information on this process, see [Get an access token](/docs/guides/implement-oauth-for-okta-serviceapp/get-access-token/).
+
+### Send a risk signal to Okta
+This procedure sends a sample risk signal payload to the Okta org.
+
+1. Call the following POST API from the Risk Integration Postman collection: **Partner: API to send RiskEvents (Auth using access token)** (`{{url}}/api/v1/risk/events/ip`). A sample payload follows, which includes two signals:
+
+    ```JSON
+    [
+     {
+       "timestamp": "2021-01-20T00:00:00.001Z",
+       "subjects": [
+          {
+            "ip": "6.7.6.7",
+            "riskLevel": "LOW",
+            "message": "none"
+          },
+          {
+            "ip": "1.1.1.1",
+            "riskLevel": "HIGH" ,
+            "message": "Detected Attack tooling and suspicious activity"
+          }
+        ]
+    }
+    ]
+    ```
+
+2. Review the request status of the API call. If the status is `202 Accepted`, the risk signals were consumed by the Okta org.
+
+For reference information on this API, see [Risk Events](/docs/reference/api/risk-events).
+
+### Confirm the response
+This procedure reviews the Admin Console's System Log to identify the risk signal information logged after calling the Risk Event API.
+
+1. Sign in to your Okta org as an administrator.
+2. From the Admin Console, navigate to the System Log (**Reports** > **System Log**).
+3. Review the log file or search for the event `security.risk.signal.consume`, which is logged when a Risk Provider sends a risk signal to Okta.
```

Comment:
"Risk Provider" is not consistently capitalized throughout. I don't have a strong opinion about capitalized or not, but it should be consistent.

Reply:
Thanks, good catch. After reviewing our style guide, standardizing on lower case references for risk providers.